### DISCUSSION TITLE
PyART Dev Call

### DISCUSSION THREAD

**Discussion by mgrover1** [ARM-DOE/pyart#1045] [Announcements]
We have started up PyART development meetings that are open to anyone interested! 

They take place every other week, on Tuesday afternoons at 9 AM Central Time, starting on September 27, 2022!

Here is the [persistent meeting agenda](https://hackmd.io/eXMjluMdRMOEscaleNWYmQ?both), and the [Microsoft Teams Link](https://teams.microsoft.com/l/meetup-join/19%3ameeting_MzJmNzhlZjAtNTc2NS00ZDNkLWFkN2QtNDMyNWEyZWI3Nzdk%40thread.v2/0?context=%7b%22Tid%22%3a%220cfca185-25f7-49e3-8ae7-704d5326e285%22%2c%22Oid%22%3a%22707efbb3-de6e-47b4-87b1-e743ad83caa7%22%7d) to join!

**Comment by mccronep** [comment]
Can I join the next meeting? Whne is the next meeting - please let me know.
Paul McCrone 
paul.mccrone@noaa.gov

**Comment by zssherman** [comment]
@mccronep Of course, thank you for your interest! Next meeting is on June 21st at 3:00PM central. The link above will work for joining the meeting.

**Comment by zssherman** [comment]
Canceling this weeks call as some of the team is out for a conference.

**Comment by mgrover1** [comment]
Update - we are having today's meeting, but will move to an earlier time in the future to accommodate those in Europe.

**Comment by kmuehlbauer** [comment]
@mgrover1 Is Teams already running?

**Comment by mgrover1** [comment]
We are sending out a new poll for availability - please use the link below to indicate what times work for you. 
- [Availability Poll](https://www.when2meet.com/?16811289-3s39r)

(@kmuehlbauer @zssherman @jrobrien91 @wolfidan @jfigui )

Also, please indicate **what times generally work well for you**, not necessarily just in two weeks. This a general "Monday mornings between 10-11 are good, etc." type poll.

**Comment by jfigui** [comment]
Hi @mgrover1 ,

I am not available the week of the 26th September.

Considering the timings proposed, which are from 16:00 to 00:00 CET, I could be mostly available from 16:00 to 18:00 on Tuesdays.

I usually work until 16:00CET.

I hope that helps,


Jordi

**Reply by mgrover1** [reply]
Hi @jfigui - even though you are not available that week, can you indicate that availability on the form for those times?

**Reply by jfigui** [reply]
Hi @mgrover1 , I did as requested

**Reply by mgrover1** [reply]
Great! Thank you so much!

**Reply by wolfidan** [reply]
Hi @mgrover1 ,

I'm really sorry but I will be in vacation that week. But starting from october I will be available.

**Reply by wolfidan** [reply]
Sorry I realized I filled up the form twice and I cannot delete the first one. Anyway I'm usually available 16h - 18:30 except on fridays.

**Comment by mgrover1** [comment]
Thanks everyone for responding! Taking a look at the responses, it looks like 9 AM Central time (1400 UTC) works well for most people. Let's plan on that! First day for this  new meeting time will be Tuesday September 27!
